consensus/hotstuff/signature/randombeacon_inspector.go

```diff
@@ -0,0 +1,117 @@
+package signature
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/engine"
+	"github.com/onflow/flow-go/model/encoding"
+	"github.com/onflow/flow-go/module/signature"
+)
+
+// randomBeaconInspector implements hotstuff.RandomBeaconInspector interface.
+// All methods of this structure are concurrency-safe.
+type randomBeaconInspector struct {
+	follower crypto.ThresholdSignatureInspector
```

Comment:
I think it would be more consistent to call this `inspector`
```suggestion
	inspector crypto.ThresholdSignatureInspector
```

Reply:
Good point